README.md

```diff
@@ -36,16 +36,19 @@ See *Background details* (below) for more details
 
 ### Creating a special jar file.
 
+Before creating the jar file you should check the Java version accepted by Solr. For example, for Solr 8.11 the recommended version of Java is hihger that 9, however higher than 11 the jar failed, then the maximun version of java you can use for generating the jas is 11.
```

Comment:
Perhaps we could just say:

For best results, compile using the same version of Java that you will use to run Solr. For Solr 8.11, JDK 11 is recommended.